### ISSUE TITLE
Please fix!!!

### ISSUE THREAD

**Issue by teacher1stgrade** [Fugiman/google-meet-grid-view#351]
I can't see all of my kids, they are lined up on top of each other.  When I try to present, the present mode take up the whole screen where I can't see my kids.  There was one that was working awesome but now you can access; 
https://chrome.google.com/webstore/detail/google-meet-grid-view-fix/dakebdbeofhmlnmjlmhjdmmjmfohiicn
Please, please, fix the grid view.  Teaching virtual is hard enough but it is worse when the technology starts messing up. From what I have seen, I'm not the only one who has requested this to be fixed.  
Thanks for all you do to hep us!

**Comment by FoxForFriends** [comment]
its been broken for a while, we got no response from author :(

**Comment by Eagle-Man** [comment]
From the Grid View (Fix) GitHub page:

> UPDATE 02/11/2021:
> It turns out that Google is testing an update of Meet that disables the Grid View extension. This is why sometimes it works and sometimes not (you see the boxes in a vertical bar). If you seriously need Grid View right now, you might consider refreshing a few times the browser tab (you should end up loading the old compatible version at a certain point).
> I am really sorry but I have to temporarily remove the extension from Chrome Web Store. In fact, updating Grid View would require a huge amount of work or may not be feasible at all. Please, remember that Google Meet introduced native grid view (up to 49 tiles).

**Comment by Eagle-Man** [comment]
@FoxForFriends There is a response from the author.  Not of the original extension, but of Grid View Fix, which I have posted above.

**Comment by Jkarhani** [comment]
> 
> 
> @FoxForFriends There is a response from the author. Not of the original extension, but of Grid View Fix, which I have posted above.

Thank you, but it's not working correctly ..

**Comment by Eagle-Man** [comment]
Yes.  Please read what I posted.  It does not and will not work.  It's pretty much dead in the water unless someone wants to put a lot of time into it on their own.

**Comment by ashokpjacob** [comment]
A temporary fix for vertical stacking
Please use this extension

[https://chrome.google.com/webstore/detail/meet-view-all/ncdkpefkincbindoojieajcilimcjaae](url)

**Comment by ashokpjacob** [comment]
A temporary fix for the problem (vertical stacking)
Please use this extension for chrome

https://chrome.google.com/webstore/detail/meet-view-all/ncdkpefkincbindoojieajcilimcjaae

**Comment by Jkarhani** [comment]
> 
> 
> A temporary fix for the problem (vertical stacking)
> Please use this extension for chrome
> 
> https://chrome.google.com/webstore/detail/meet-view-all/ncdkpefkincbindoojieajcilimcjaae

Thank you .. It's working sooo far..

**Comment by lisdab** [comment]
Same thing. When you present. Only 8 students will be seen on the screen. Anyone knows how to see all the students on screen? Thanks

**Comment by lisdab** [comment]
> I can't see all of my kids, they are lined up on top of each other. When I try to present, the present mode take up the whole screen where I can't see my kids. There was one that was working awesome but now you can access;
> https://chrome.google.com/webstore/detail/google-meet-grid-view-fix/dakebdbeofhmlnmjlmhjdmmjmfohiicn
> Please, please, fix the grid view. Teaching virtual is hard enough but it is worse when the technology starts messing up. From what I have seen, I'm not the only one who has requested this to be fixed.
> Thanks for all you do to hep us!

Do you find a way to see all the students on the screen? Same thing is happening, when I present it covers the screen and cant see my students. Please help!

**Comment by ashokpjacob** [comment]
> 
> 
> > I can't see all of my kids, they are lined up on top of each other. When I try to present, the present mode take up the whole screen where I can't see my kids. There was one that was working awesome but now you can access;
> > https://chrome.google.com/webstore/detail/google-meet-grid-view-fix/dakebdbeofhmlnmjlmhjdmmjmfohiicn
> > Please, please, fix the grid view. Teaching virtual is hard enough but it is worse when the technology starts messing up. From what I have seen, I'm not the only one who has requested this to be fixed.
> > Thanks for all you do to hep us!
> 
> Do you find a way to see all the students on the screen? Same thing is happening, when I present it covers the screen and cant see my students. Please help!



Yes you can see all the childern while presenting (a temporary fix)
Please use this extension for chrome

https://chrome.google.com/webstore/detail/meet-view-all/ncdkpefkincbindoojieajcilimcjaae

**Comment by Jkarhani** [comment]
here what I had: 
7 are not showing, only 14 are showing.
Any tips @ashokpjacob ? do i have to go over change layout then tiled?
[https://drive.google.com/file/d/1P_t2yyLLmKCnr6pdA5fbzm7PssIC5Mox/view?usp=sharing](url)

**Comment by lisdab** [comment]
> > > I can't see all of my kids, they are lined up on top of each other. When I try to present, the present mode take up the whole screen where I can't see my kids. There was one that was working awesome but now you can access;
> > > https://chrome.google.com/webstore/detail/google-meet-grid-view-fix/dakebdbeofhmlnmjlmhjdmmjmfohiicn
> > > Please, please, fix the grid view. Teaching virtual is hard enough but it is worse when the technology starts messing up. From what I have seen, I'm not the only one who has requested this to be fixed.
> > > Thanks for all you do to hep us!
> > 
> > 
> > Do you find a way to see all the students on the screen? Same thing is happening, when I present it covers the screen and cant see my students. Please help!
> 
> Yes you can see all the childern while presenting (a temporary fix)
> Please use this extension for chrome
> 
> https://chrome.google.com/webstore/detail/meet-view-all/ncdkpefkincbindoojieajcilimcjaae

I tried all the GRID FOR GOOGLE MEET NO LUCK. THEY ALL DID NOT WORK AND I EVEN USED ONLY THE TILES OF GOOGLE NOT WORKING. Hope you can fix soonest. I have 25 in class and only 8 showing :( Help pls community :)

**Comment by Eagle-Man** [comment]
@lisdab This is because all of the extensions (except maybe ashok's?) are based on the same code, which is massively broken by Google's Meet code update.

Google's Tiled View should work up to 49 people.  Ensure you do not have any grid view-like extensions active which might disrupt it (uninstall them all to be sure).  Remember that with Tiled View, you may have to adjust the slider to show the right number of videos.

**Comment by lisdab** [comment]
> @lisdab This is because all of the extensions (except maybe ashok's?) are based on the same code, which is massively broken by Google's Meet code update.
> 
> Google's Tiled View should work up to 49 people. Ensure you do not have any grid view-like extensions active which might disrupt it (uninstall them all to be sure). Remember that with Tiled View, you may have to adjust the slider to show the right number of videos.

Thank you so much for your response. The whole school's problem is this at the moment. I've tried everything :( Removed all the extensions that might affect the tiled of google meet but to no avail. I tried the tile up to 49, it's not working WHEN I PRESENT TO MY STUDENTS, very frustrating not to see them all :( I tried installing all the grid too but they don't all work. We are suffering as we are looking at the list only to call our students but unable to see all of them. :(

**Comment by Jkarhani** [comment]
> 
> 
> > @lisdab This is because all of the extensions (except maybe ashok's?) are based on the same code, which is massively broken by Google's Meet code update.
> > Google's Tiled View should work up to 49 people. Ensure you do not have any grid view-like extensions active which might disrupt it (uninstall them all to be sure). Remember that with Tiled View, you may have to adjust the slider to show the right number of videos.
> 
> Thank you so much for your response. The whole school's problem is this at the moment. I've tried everything :( Removed all the extensions that might affect the tiled of google meet but to no avail. I tried the tile up to 49, it's not working WHEN I PRESENT TO MY STUDENTS, very frustrating not to see them all :( I tried installing all the grid too but they don't all work. We are suffering as we are looking at the list only to call our students but unable to see all of them. :(

Exactly, same here.

**Comment by lisdab** [comment]
> > > @lisdab This is because all of the extensions (except maybe ashok's?) are based on the same code, which is massively broken by Google's Meet code update.
> > > Google's Tiled View should work up to 49 people. Ensure you do not have any grid view-like extensions active which might disrupt it (uninstall them all to be sure). Remember that with Tiled View, you may have to adjust the slider to show the right number of videos.
> > 
> > 
> > Thank you so much for your response. The whole school's problem is this at the moment. I've tried everything :( Removed all the extensions that might affect the tiled of google meet but to no avail. I tried the tile up to 49, it's not working WHEN I PRESENT TO MY STUDENTS, very frustrating not to see them all :( I tried installing all the grid too but they don't all work. We are suffering as we are looking at the list only to call our students but unable to see all of them. :(
> 
> Exactly, same here.

Hope it'll be fixed soon otherwise we will accept not seeing students while discussing :( very sad

**Comment by Eagle-Man** [comment]
Oh!  That’s different.  What Google fails to mention in any noticeable way is that when presenting or having someone present, there is no expectation of full numbers of people shown.  At that point, the number of people shown is based on the Resolution of the OS and the zoom level of the OS and the browser.  Higher resolution means more videos.

I have my laptop at max resolution (1900+) and I reduced Windows’ default zoom of 150% down to 100%, and I can still see 49 when presenting, but only 24 when others present.  It could be more if I zoom out in the browser, too.

Out of curiosity, when you present, do you see people along the top or the side?  I heard the former but haven’t seen it, so it may be something they’re testing.  But I hear it shows fewer people when it’s along the top.

**Comment by ashokpjacob** [comment]
@Jkarhani @teacher1stgrade @Eagle-Man 

This extension :-

https://chrome.google.com/webstore/detail/meet-view-all/ncdkpefkincbindoojieajcilimcjaae 

To make it work:

1. Ensure that no other grid extensions are active.
2. Change layout to **Tiled View**.
3. Adjust the slider to Maximum number of videos (49)
4. Set Zoom level of Browser to 75% (use the Ctrl key and  “-”)

This Works even when you present the screen

**Comment by Jkarhani** [comment]
@ashokpjacob sir, this is exactly what we did..it's better than nothing but not like before when presenting..

**Comment by lisdab** [comment]
> Oh! That’s different. What Google fails to mention in any noticeable way is that when presenting or having someone present, there is no expectation of full numbers of people shown. At that point, the number of people shown is based on the Resolution of the OS and the zoom level of the OS and the browser. Higher resolution means more videos. I have my laptop at max resolution (1900+) and I reduced Windows’ default zoom of 150% down to 100%, and I can still see 49 when presenting, but only 24 when others present. It could be more if I zoom out in the browser, too. Out of curiosity, when you present, do you see people along the top or the side? I heard the former but haven’t seen it, so it may be something they’re testing. But I hear it shows fewer people when it’s along the top.
> […](#)
> On Feb 22, 2021, at 1:56 AM, lisdab ***@***.***> wrote: ﻿ @lisdab This is because all of the extensions (except maybe ashok's?) are based on the same code, which is massively broken by Google's Meet code update. Google's Tiled View should work up to 49 people. Ensure you do not have any grid view-like extensions active which might disrupt it (uninstall them all to be sure). Remember that with Tiled View, you may have to adjust the slider to show the right number of videos. Thank you so much for your response. The whole school's problem is this at the moment. I've tried everything :( Removed all the extensions that might affect the tiled of google meet but to no avail. I tried the tile up to 49, it's not working WHEN I PRESENT TO MY STUDENTS, very frustrating not to see them all :( I tried installing all the grid too but they don't all work. We are suffering as we are looking at the list only to call our students but unable to see all of them. :( — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

I tried this, yes it worked I could see all of them but the interface became so small. this will be a temporary fix, but the chat and everything could not be read anymore :(

**Comment by lisdab** [comment]
> > Oh! That’s different. What Google fails to mention in any noticeable way is that when presenting or having someone present, there is no expectation of full numbers of people shown. At that point, the number of people shown is based on the Resolution of the OS and the zoom level of the OS and the browser. Higher resolution means more videos. I have my laptop at max resolution (1900+) and I reduced Windows’ default zoom of 150% down to 100%, and I can still see 49 when presenting, but only 24 when others present. It could be more if I zoom out in the browser, too. Out of curiosity, when you present, do you see people along the top or the side? I heard the former but haven’t seen it, so it may be something they’re testing. But I hear it shows fewer people when it’s along the top.
> > […](#)
> > On Feb 22, 2021, at 1:56 AM, lisdab _**@**_.***> wrote: ﻿ @lisdab This is because all of the extensions (except maybe ashok's?) are based on the same code, which is massively broken by Google's Meet code update. Google's Tiled View should work up to 49 people. Ensure you do not have any grid view-like extensions active which might disrupt it (uninstall them all to be sure). Remember that with Tiled View, you may have to adjust the slider to show the right number of videos. Thank you so much for your response. The whole school's problem is this at the moment. I've tried everything :( Removed all the extensions that might affect the tiled of google meet but to no avail. I tried the tile up to 49, it's not working WHEN I PRESENT TO MY STUDENTS, very frustrating not to see them all :( I tried installing all the grid too but they don't all work. We are suffering as we are looking at the list only to call our students but unable to see all of them. :( — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.
> 
> I tried this, yes it worked I could see all of them but the interface became so small. this will be a temporary fix, but the chat and everything could not be read anymore :(

When I present I can see only 8 students in a normal zoom

**Comment by Craftzman7** [comment]
I mean if you absolutely need it you could try and make a pull request
